iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -805,4 +802,19 @@ private boolean hasBundledSDK(EventProcessingRequest processingRequest) {
                 integrationAttributes.getOrDefault("Iterable.sdkVersion", null) != null;
     }
 
+    private void handleIterableResponse(Response<IterableApiResponse> iterableResponse) throws IOException {
+        // consider OkHttp Interceptor if this will be added to all methods
+        if (iterableResponse.isSuccessful() && !iterableResponse.body().isSuccess()) {
+            throw new IOException(iterableResponse.body().toString());
+        } else if (!iterableResponse.isSuccessful()) {
+            if (iterableResponse.code() == 429 ) {
+                IterableLambdaResponse lambdaResponse = new IterableLambdaResponse();
```

Comment:
This doesn't seem to do anything. I think you'll need to pull this into `IterableLambdaEndpoint`